src/main/java/edu/harvard/iq/dataverse/SettingsWrapper.java

```diff
@@ -360,36 +360,39 @@ public boolean isEmbargoAllowed() {
     
     public void validateEmbargoDate(FacesContext context, UIComponent component, Object value)
             throws ValidatorException {
-        UIComponent cb = component.findComponent("embargoCheckbox");
-        UIInput endComponent = (UIInput) cb;
-        boolean removedState = false;
-        if (endComponent != null) {
-            try {
-                removedState = (Boolean) endComponent.getSubmittedValue();
-            } catch (NullPointerException npe) {
-                // Do nothing - checkbox is not being shown (and is therefore not checked)
+        if (isEmbargoAllowed()) {
```

Comment:
The only change is adding this if (and then indenting the whole method so github shows a big change) which stops validation from running when embargoes are not enabled (and maxEmbargoDate is then null).